frontend/src/app/sessions/session/session.component.css

```diff
@@ -7,6 +7,10 @@
   height: calc(100vh - 2vh - 65px - 110px);
 }
 
+.heigh-fullscreen {
+  height: calc(100vh - 2.6vh);
```

Comment:
Where does the `2.6vh` come from?

Reply:
It was more or less testing. Using a value of `2vh` led to a scrolling bar on the side while a value of `3vh` resulted in some "unnecessary" white area below the sessions

Reply:
You applied the class to the wrong element. If you apply it to the div element, which contains iframe & header, you can use `h-full`. You did just apply it to the iframe and removed the approximate size of the header. This isn't a good style as the header size has untracked dependencies in other CSS classes like this one.